### DISCUSSION TITLE
0:0  error  Parsing error: Cannot read property 'uid' of undefined. When using sls deploy.

### DISCUSSION THREAD

**Discussion by emon-gain** [serverless/serverless#10444] [Q&A]
### Are you certain it's a bug?

- [X] Yes, it looks like a bug

### Is the issue caused by a plugin?

- [X] It is not a plugin issue

### Are you using the latest version?

- [X] Yes, I'm using the latest version

### Is there an existing issue for this?

- [X] I have searched existing issues, it hasn't been reported yet

### Issue description

When I am using github actions for deploying my code to lambda using sls deploy it shows me this error. I could not found the solution of these error.

### Service configuration (serverless.yml) content

```yaml
N/A
```

### Command name and used flags

sls deploy

### Command output

```shell
Serverless: Bundling with Webpack...
ERROR in /home/runner/work/***/src/handler.js
  0:0  error  Parsing error: Cannot read property 'uid' of undefined

✖ 1 problem (1 error, 0 warnings)

webpack compiled with 1 error
 
 Serverless Error ----------------------------------------
 
  Webpack compilation error, see stats above
 
  Get Support --------------------------------------------
     Docs:          docs.serverless.com
     Bugs:          github.com/serverless/serverless/issues
     Issues:        forum.serverless.com
 
  Your Environment Information ---------------------------
     Operating System:          linux
     Node Version:              14.18.2
     Framework Version:         2.70.0
     Plugin Version:            5.5.2
     SDK Version:               4.3.0
     Components Version:        3.18.1
 
Error: Process completed with exit code 1.
```

### Environment information

```shell
APP_ENVIRONMENT: ***
    SCRIPT_NAME: ***
    KMS_CTX: ***
    API_KEY: ***
```

**Comment by pgrzesik** [comment]
I'm going to convert it to a discussion as it seems to be a problem with your function's code that causes crash during webpack compilation - could you provide a small reproducible example?

**Comment by eurafaeldecarvalho** [comment]
I have the same problem here. I managed to upload the same code on Thursday but it's been giving this error now

**Reply by pgrzesik** [reply]
Hello @rafaelgdn - could you share a bit more details about your setup? Do you happen to have a small reproducible example that I could check out on my side?

**Comment by bckwcs** [comment]
@emon-gain @rafaelgdn updating to latest serverless fixed my issue
npm i -g serverless

**Comment by washingtong33k** [comment]
The problem seems to be with graceful-fs that was upgraded from 4.2.6 to 4.2.9

**Reply by OnTheMike** [reply]
I'm also seeing this error related to the graceful-fs. Upgrading serverless unfortunately didn't resolve the issue for me. Our issue seemed to be caused by serverless-plugin-include-dependencies, disabling that resolved the issue with the uid-error.

**Reply by yuvalyacoby** [reply]
had the same issue where it broke on graceful-fs from include-dependencies. Upgrading serverless globally did help though

**Comment by Ionut-Milas** [comment]
Any updates on this ?

**Reply by pgrzesik** [reply]
Are you still experiencing issues? Did you reinstall serverless along with all its dependencies?

**Comment by bitsofinfo** [comment]
experiencing the same

**Reply by pgrzesik** [reply]
Hello @bitsofinfo - what version are you using? Did you ensure to reinstall dependencies by reinstalling serverless?

**Reply by bitsofinfo** [reply]
2.54.0 installed globally, 2.70 installed locally in node modules. Only after forcing the global install to be the latest did the issue go away.
